### DISCUSSION TITLE
Updating to version 0.12.1

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#6858] [Ask A Question]
I just upgraded to Frigate 0.12.1 by changing editing my ```frigate.yaml``` file to read: ```image: ghcr.io/blakeblackshear/frigate:0.12.1```.  Everything seems to be running smoothly with no other changes to my ```config.yml``` file.

Thank you to all involved in making this incredible software. My house is in a city with a terrible reputation for crime and having Frigate monitor my cameras makes me sleep more easily at night, and definitely gives me peace of mind when I travel.

**Comment by yuckypants** [comment]
I just wanted to reply to say how awesome you are. The fact that this information isn't found ANYWHERE is astounding. I'm shocked with how thorough the documentation is, it's still so largely incomplete and makes a lot of assumptions. Updating installations should be part of the installation section, and not finding out about it like this. 

Anyway, just wanted to say thanks to you, @Cold-Lemonade.
